mkt/account/serializers.py

```diff
@@ -137,3 +138,12 @@ class UserSerializer(AccountSerializer):
     class Meta:
         model = UserProfile
         fields = ('display_name', 'resource_uri')
+
+
+class GroupsSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(source='pk', required=False, read_only=False)
```

Comment:
Do you need this ? It's a `ModelSerializer`, it should do that automatically, shouldn't it ?